Packs/Tessian/Integrations/Tessian/Tessian.py

```diff
@@ -0,0 +1,336 @@
+    event_id = args.get('event_id', None)
+
+    if event_id is None:
+        raise ValueError('Event ID is required')
+
+    results = client.delete_from_inbox(event_id)
+    results["event_id"] = event_id
+
+    markdown = '# Delete from Inbox Action\n'
+    markdown += f'## Event ID: {event_id}\n'
+    markdown += f'## Number of Delete Actions Successfully Initiated: {results.get("number_of_actions_succeeded")}\n'
+    failure_count = results.get("number_of_actions_attempted", 0) - results.get("number_of_actions_succeeded", 0)
+    markdown += f'## Number of Delete Actions Failed: {failure_count}\n'
+
+    if results.get("number_of_actions_attempted") != results.get("number_of_actions_succeeded"):
+        failures = [
+            {
+                "Recipient": failure.get("user_address"),
+                "Error": failure.get("error")
+            } for failure in results.get("results", []) if failure.get("error") is not None
+        ]
+        markdown += tableToMarkdown(
+            name="Errors",
+            t=failures,
+            headers=['Recipient', 'Error'],
+        )
+
+    return CommandResults(
+        outputs_prefix='Tessian.DeleteFromInboxOutput',
+        outputs_key_field='event_id',
+        outputs=results,
+        raw_response=results,
+        readable_output=markdown,
+    )
+
+
+def test_module(client: Client) -> str:  #  pragma: no cover
+    """
+    Tests API connectivity and authentication'
+    Returning 'ok' indicates that connection to the service is successful.
+    Raises exceptions if something goes wrong.
+    """
+
+    try:
+        response = client.list_events(2, None, None)
+
+        success = demisto.get(response, 'checkpoint')
+        if success is None:
+            return f'Unexpected result from the service: expected checkpoint to be a string, response={str(response)}'
+
+        return 'ok'
+    except Exception as e:
+        exception_text = str(e).lower()
+        if 'forbidden' in exception_text or 'authorization' in exception_text:
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:  #  pragma: no cover
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    # get the service API url
+    params = demisto.params()
+    args = demisto.args()
+    base_url = format_url(params.get('url'))
+    api_key = params.get('api_key')
+
+    # if your Client class inherits from BaseClient, SSL verification is
+    # handled out of the box by it, just pass ``verify_certificate`` to
+    # the Client constructor
+    verify_certificate = not demisto.params().get('insecure', False)
+
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
+    proxy = demisto.params().get('proxy', False)
```

Comment:
```suggestion
    verify_certificate = not demisto.params().get('insecure', False)

    # if your Client class inherits from BaseClient, system proxy is handled
    # out of the box by it, just pass ``proxy`` to the Client constructor
    proxy = demisto.params().get('proxy', False)
```
please add the `insecure` and `proxy` params to the yml as well